quinn-proto/src/config/mod.rs

```diff
@@ -159,6 +162,25 @@ impl EndpointConfig {
         self.rng_seed = seed;
         self
     }
+
+    /// Minimum size of the first Initial packet. You should NEVER consider calling this function
+    /// under any circumstances.
+    ///
+    /// <div class="warning">
+    ///
+    /// According to the quinn specification, this value MUST BE 1200. Modifying this value may have
+    /// unexpected consequences.
```

Comment:
Quinn doesn't have a specification. Quinn is an implementation of the QUIC specification. Changing this value makes Quinn's behavior non-conformant with the QUIC specification. For this documentation to be actionable, it should outline *what* the consequences are, not just handwave them as "unexpected".